### ISSUE TITLE
epub continue on last read page

### ISSUE THREAD

**Issue by richukuttan** [elias-sundqvist/obsidian-annotator#99]
Hi,

I think the pdf reader auto-opens pdfs to the last read page. Is there any way to implement the same for epub? If the reader does not support the same out-of-the-box, maybe you can store a "last read page" dataview variable in the markdown, and access that while loading the epub.

I generally store 300+ page books, and it gets really annoying trying to find the page I last read.

Thanks for the plugin. I can safely say that my workflow would have been much clunkier without this!

**Comment by bagameow** [comment]
+1  I think this is a must.

**Comment by warmbupt** [comment]
I'd like to know if there is any plan for this feature?

**Comment by salvlox** [comment]
Yes please, I though that was already included and I discovered today that it wasnt
